services/background-worker/src/Wire/BackgroundWorker/Options.hs

```diff
@@ -31,7 +30,9 @@ data BackendNotificationsConfig = BackendNotificationsConfig
     -- | Upper limit on amount of time (in microseconds) to wait before retrying
     -- any notification. This exists to ensure that exponential back-off doesn't
     -- cause wait times to be very big.
-    pushBackoffMaxWait :: Int
+    pushBackoffMaxWait :: Int,
+    -- | Number of seconds between two calls to `getRemoteDomains`.
+    remotesRefreshInterval :: Maybe Double
```

Comment:
Instead of using `Double` can we just take number of milliseconds? We won't need rounding like this. I think we can safely assume people running wire are capable of understanding different units of time than seconds.
This also makes it more in line with the config one line above.

Reply:
i get the last point, but i just think doubles are more ergonomic.  anyway i'll align it with the rest.